File: modulo1/segmento2/apunte1/Ejercicio1.java
import java.util.Scanner;
public class Ejercicio1
{
	public static void main(String args[])
	{
		Scanner keyboard = new Scanner(System.in);

		System.out.print("¿Cual es el barrio de la comisaria?: ");
		String location = keyboard.nextLine();

		System.out.print("¿Cual es la direccion?: ");
		String address = keyboard.nextLine();

		System.out.print("¿Entre que calle esta?, Escriba el nombre de la primera: ");
		String street1 = keyboard.nextLine();

		System.out.print("Cual es el nombre de la segunda calle?: ");
		String street2 = keyboard.nextLine();

		System.out.println("La comisria mas cercana esta en la calle: " + address + " Entre la calle: " + street1 + " y la calle: " + street2 + " Ubicado en el barrio de: " + location);

	
	}

}

File: modulo1/segmento2/apunte1/ejercicio1/ejercicio1.web/ejercicio1.js
function saludar(){
	const barrio = prompt('Cual es el barrio de la comisaria mas cercana');
		alert(`El barrio de la comisaria esta en:  ${barrio}`);
	
	const direccion = prompt ('Cual es la direccion');
		alert(`En la calle:  ${direccion}`);

	const calle1 = prompt ('Esta entre la calle:');
		alert(`Entre la calle:  ${calle1}`);

	const calle2 = prompt ('Y la calle ');
		alert(`Y la calle: ${calle2}`);

		alert('la comisaria esta en la calle: ' + direccion + ' entre la calle: ' + calle1 + ' y la calle: ' + calle2 + ' en el barrio de: ' + barrio);
}
